TestUrdfImporter/Assets/test.txt

```diff
@@ -0,0 +1,20 @@
+        <sensor name='my_camera' type='camera'>
+          <camera>
+            <save enabled="true">
+              <path>/tmp/camera_save_tutorial</path>
+            </save>
+            <horizontal_fov>1.047</horizontal_fov>
+            <image>
+              <width>1920</width>
+              <height>1080</height>
+            </image>
+            <clip>
+              <near>0.1</near>
+              <far>100</far>
+            </clip>
+          </camera>
+          <always_on>1</always_on>
+          <update_rate>30</update_rate>
+          <plugin name='test_plugin' filename='test_filename'>
+          </plugin>
+        </sensor>
```

Comment:
This file needs to be moved to  `com.unity.robotics.urdf-importer/Tests`  , probably under `/Resources  `